src/types.ts

```diff
@@ -206,6 +206,7 @@ export interface Logger {
 export interface UserData {
   name: string;
   email: string;
+  date?: Date;
```

Comment:
I don't think I follow why date is part of the UserData as opposed to the commit